### DISCUSSION TITLE
Version 2.9.0 coming with three new features 🌟

### DISCUSSION THREAD

**Discussion by muety** [muety/wakapi#530] [Announcements]
In the last couple of day, I have had some time to finally work a bit more intensively on Wakapi again. Today, I'm happy to announce the release of version 2.9.0 :raised_hands:. It's our biggest release in a while and comes with three new features.

## Activity Chart (#12)
Inspired by the contribution chart on peoples' GitHub profiles, Wakapi not generates a calendar-like visualization of your per-day coding intensity over the past year.

![Screenshot from 2023-10-03 20-07-53](https://github.com/muety/wakapi/assets/8978843/6e13ab82-50c7-4ef9-9b3b-fdcb9501dbbf)

## More Filtering (#511)
So far, it was possible to filter your coding summary by time. With this release, you'll additionally get the chance to apply project-, language-, machine- and label filters to your dashboard.

![Screenshot from 2023-10-03 20-07-22](https://github.com/muety/wakapi/assets/8978843/08137c28-0238-463f-bbec-65412373a7e6)

## Project Overview (#302)
Last, but not least, you'll now get an overview of all projects you've ever worked on while using Wakapi. 

![Screenshot from 2023-10-03 20-21-50](https://github.com/muety/wakapi/assets/8978843/78e47161-fd8d-4167-b71e-3a7190f66b58)

With big changes like these, unfortunately, there is also a risk to have introduced new bugs into the code. Please bear with us, report anything that looks like a bug and we'll do our best to get it fixed as as soon as possible.

**Comment by Kovah** [comment]
It's so cool how Wakapi is evolving. Great work!

**Comment by oasido** [comment]
Great work! Loving the new update.
